fighters/link/src/acmd/aerials.rs

```diff
@@ -197,21 +153,17 @@ unsafe fn attack_air_hi(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.600);
-    }
+    FT_MOTION_RATE(fighter, 0.6);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`